### PR TITLE
Add option to debug BSP connection.

### PR DESCRIPTION
**Pull request by romanowski**

### CODE REVIEW

File: modules/cli/src/main/scala/scala/cli/commands/SharedJvmOptions.scala [hunk 1]
```diff
@@ -47,7 +47,13 @@ final case class SharedJvmOptions(
   @HelpMessage("Javac options")
   @Name("javacOpt")
   @Hidden
-    javacOption: List[String] = Nil
+    javacOption: List[String] = Nil,
+
+  @Group("Java")
+  @HelpMessage("Port for BSP debugging")
+  @Name("bspDebugPort")
```
Comment (alexarchambault) [inline modules/cli/src/main/scala/scala/cli/commands/SharedJvmOptions.scala]: Seems this makes the duplicated option check fail (this adds the same name as the variable).
```suggestion
```

**Review by alexarchambault — COMMENTED** [review]
Apart from the comment, and the reference doc that needs to be updated (addressing both should make the CI pass), LGTM!

File: modules/cli/src/main/scala/scala/cli/commands/SharedJvmOptions.scala (hunk 1 above)
Comment (romanowski) [reply modules/cli/src/main/scala/scala/cli/commands/SharedJvmOptions.scala]: Good point, I was copy-pasting :)

**Review by alexarchambault — APPROVED** [review]

**Comment by alexarchambault** [comment]
@romanowski Seems wrong stuff (bad rebase?) were pushed.

**Review by Gedochao — APPROVED** [review]

**Pull request merged**